fe/fe-core/src/main/java/com/starrocks/cluster/Cluster.java

```diff
@@ -90,10 +64,8 @@ public void write(DataOutput out) throws IOException {
         out.writeLong(id);
         Text.writeString(out, name);
 
-        out.writeLong(backendIdSet.size());
-        for (Long id : backendIdSet) {
-            out.writeLong(id);
-        }
+        // For compatible for backendIdSet
```

Comment:
We should make sure the backendIdSet is never used in all versions. Or the result will be empty when the cluster is downgrade.